src/Api/Billing/Public/Models/SecretsManagerSubscriptionUpdateRequestModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Bit.Core.Entities;
+using Bit.Core.Models.Business;
+
+namespace Bit.Api.Billing.Public.Models;
+
+public class SecretsManagerSubscriptionUpdateRequestModel
+{
+    [Required]
+    public int SeatAdjustment { get; set; }
+    public int? MaxAutoscaleSeats { get; set; }
+    public int ServiceAccountAdjustment { get; set; }
+    public int? MaxAutoscaleServiceAccounts { get; set; }
```

Comment:
I don't like that seats and service account values use deltas ("adjustment" amounts) whereas max autoscale values use totals. We do it in our private API and I don't like it there either. I think it's confusing to have similar values treated differently, and "adjustment" values are not idempotent (i.e. if you accidentally send it multiple times you'll have multiple charges).

I suggest that all properties use totals, i.e.
```suggestion
    public int Seats { get; set; }
    public int? MaxAutoscaleSeats { get; set; }
    public int ServiceAccounts { get; set; }
    public int? MaxAutoscaleServiceAccounts { get; set; }
```

However, this is just my personal bugbear, has there been any input from Product or the CS Integration team as to their preferred API design?

Note: I have not done a full review, just flagging this issue.

Reply:
I agree with @eliykat, if have the opportunity to keep these consistent in a new controller, I think it's worth doing so.